### ISSUE TITLE
Type Facades for Protractor

### ISSUE THREAD

**Issue by mfoody** [greencatsoft/scalajs-angular#95]
Do you have interest in adding type facades for Protractor? I'd be happy to submit a pull request.

**Comment by mysticfall** [comment]
Of course, I do :) Actually, test related API has been one of the most important bits which is still missing in this project.

Personally, I'm using ScalaTest to write unit tests for a project using this library, so I'd like to see better integration with the framework first.

It definitely won't hurt to have another option to write unit tests, and we may even be able to make some sort of a bridge between Protractor and ScalaTest, in which scenario, such type facades could serve as a starting point, in my opinion.

So, I'd like to encourage you to give it a go, and I'd appreciate if you could make a pull request.

Thanks!

**Comment by mfoody** [comment]
Sounds good. We unfortunately started our project writing the production code in Coffeescript. However we just switched our end to end Protractor tests to ScalaJS and have had very positive results. I'll work on getting you a pull request.
